pkg/web/interfaces.go

```diff
@@ -30,6 +30,21 @@ type OpenFGAClientInterface interface {
 	ReadTuples(context.Context, string, string, string, string) (*openfga.ReadResponse, error)
 }
 
+type AuthorizerClientInterface interface {
+	ListObjects(context.Context, string, string, string) ([]string, error)
+	Check(context.Context, string, string, string, ...ofga.Tuple) (bool, error)
+	CreateClient(context.Context, string) error
+	DeleteClient(context.Context, string) error
+	CreateIdentity(context.Context, string) error
+	DeleteIdentity(context.Context, string) error
+	CreateProvider(context.Context, string) error
+	DeleteProvider(context.Context, string) error
+	CreateRule(context.Context, string) error
+	DeleteRule(context.Context, string) error
+	CreateSchema(context.Context, string) error
+	DeleteSchema(context.Context, string) error
+}
+
```

Comment:
ideally we move this into each package where each `AuthorizerClientInterface` implements a subset of these being used

aka `identities` pkg will have

```
type AuthorizerClientInterface interface {
	CreateIdentity(context.Context, string) error
	DeleteIdentity(context.Context, string) error
}
```

while  `idps` pkg will have

```
type AuthorizerClientInterface interface {
	CreateProvider(context.Context, string) error
	DeleteProvider(context.Context, string) error
}
```

in here we can simply pass the pointer to the Authorizer

Reply:
That's already implemented (eg https://github.com/canonical/identity-platform-admin-ui/pull/349/files#diff-ecd1991c6a0e8a942b9f3669dff799d8588c817d694555198c6caa0f9986ea58).